inc/plugin.class.php

```diff
@@ -1699,12 +1712,16 @@ static function getState($state = 0) {
    public static function getPlugins() {
       global $GLPI_CACHE;
 
-      if (!$GLPI_CACHE->has('plugins')) {
-         $self = new self();
-         $self->init();
+      if ($GLPI_CACHE instanceof CacheInterface && $GLPI_CACHE->has('plugins')) {
```

Comment:
I put back old logic, as autoinit leads to inifinite loops when method is called during the init process.
I added some more check to be sure that returned data is always an array.